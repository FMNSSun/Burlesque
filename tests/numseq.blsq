,254rz^^2B!)<-2B!$$p^