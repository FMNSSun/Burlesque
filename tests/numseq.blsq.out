0
0
3
0
5
0
5
0
9
0
15
6
15
6
9
0
17
0
27
10
17
0
27
10
27
10
17
0
27
10
17
0
33
0
51
18
45
12
63
30
45
12
63
30
33
0
51
18
51
18
33
0
63
30
45
12
63
30
45
12
51
18
33
0
65
0
99
34
85
20
119
54
65
0
99
34
85
20
119
54
85
20
119
54
65
0
99
34
85
20
119
54
65
0
99
34
99
34
65
0
119
54
85
20
99
34
65
0
119
54
85
20
119
54
85
20
99
34
65
0
119
54
85
20
99
34
65
0
129
0
195
66
165
36
231
102
153
24
219
90
189
60
255
126
153
24
219
90
189
60
255
126
129
0
195
66
165
36
231
102
165
36
231
102
129
0
195
66
189
60
255
126
153
24
219
90
189
60
255
126
153
24
219
90
165
36
231
102
129
0
195
66
195
66
129
0
231
102
165
36
219
90
153
24
255
126
189
60
219
90
153
24
255
126
189
60
195
66
129
0
231
102
165
36
231
102
165
36
195
66
129
0
255
126
189
60
219
90
153
24
255
126
189
60
219
90
153
24
231
102
165
36
195
66
129